### PR TITLE
Feature/#104 avatars colors

### LINKED ISSUE #104: Extend ModeModal with colors and avatars

### PR DESCRIPTION
**Pull request by DominikNowak**
closes #104

### CODE REVIEW

File: src/app/components/ModeModal.ts [hunk 1]
```diff
@@ -2,10 +2,10 @@ import { BasicModal } from './BasicModal';
 import { Render } from '../utils/Render';
 import { PlayerDTO } from '~src/Interfaces/PlayerDTOInterface';
 import { CallbackOneParam } from '~src/Interfaces/CallbackOneParamInterface';
-
+import { Avatars } from '~src/Enums/AvatarsEnum';
```
Comment (vieraboschkova) [inline src/app/components/ModeModal.ts:5]: Check if you should remove tildas from paths

File: src/app/components/ModeModal.ts [hunk 2]
```diff
@@ -155,63 +151,150 @@ export class ModeModal extends BasicModal {
...
+          playersData.push({
+            name: '',
+            path: Avatars.FARMER1,
+            color: Colors.GREEN,
+          } as PlayerDTO);
+          playersData[index].name =
             value.trim().length > 0
               ? value
-              : `Janush ${playersData.length + 1}`;
+              : `Janush ${playersData.length}`;
```
Comment (vieraboschkova) [inline src/app/components/ModeModal.ts:286]: Perhaps the default names should match, or be given in placeholder? (for example 'type nickname here' and then Player1, Player2 on the board, or 'type nickname here, ex.: Janush1')

**Review by vieraboschkova — APPROVED** [review]

**Review by SebastianBabinski1 — APPROVED** [review]
Looks good

File: src/app/components/ModeModal.ts (hunk 2 above)
Comment (DominikNowak) [reply src/app/components/ModeModal.ts:286]: Placeholder will be 'type nickname here' according to figma project

File: src/app/components/ModeModal.ts (hunk 1 above)
Comment (DominikNowak) [reply src/app/components/ModeModal.ts:5]: It's from autoimport. Is it something wrong with it?

**Pull request merged**